targets/tizenrt-artik05x/configs/jerryscript/Make.defs

```diff
@@ -1,5 +1,37 @@
+###########################################################################
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###########################################################################
+#
+# Copyright 2017 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
```

Comment:
@akosthekiss First, thank you again for explanation. By the way, I did not add the header with a `Copyright 2017 Samsung Electronics All Rights Reserved`. The header came from [the base file](https://github.com/Samsung/TizenRT/blob/master/build/configs/artik053/tash/Make.defs) in TizenRT. I just added JS Foundation license only. The result was my first commit like following:

JS Foundation license (Apache2 : for conforming jerryscript license rule)
Samsung license (Apache2 : from TizenRT)
NuttX license (BSD-like : from NuttX)